bin/convert-translations-from-xml-to-json.js

```diff
@@ -1,42 +1,48 @@
 const converter = require('xml-js')
 const path = require('path')
 const fs = require('fs')
+const LOCALES_DIR = path.join(__dirname, '..', '_locales')
 
-var LOCALES_DIR = path.join(__dirname, '..', '_locales')
-console.log('Converting translations from xml to json...')
-fs.readdir(LOCALES_DIR, (err, filenames) => {
+const start = Date.now()
+process.stdout.write('Converting translations from xml to json... ')
+fs.readdir(LOCALES_DIR, (err, files) => {
   if (err) throw err
-  filenames.filter((filename) => path.extname(filename) === '.xml').forEach(xmlToJson)
+  const xmlFiles = files.filter(filename => path.extname(filename) === '.xml')
+  xmlFiles
+    .map(filename => path.resolve(LOCALES_DIR, filename))
+    .forEach(xmlToJson)
```

Comment:
I made a blunder here, e.g. `.forEach(filename => xmlToJson)`, which doesn't work so well :wink:

Reply:
right, I missed that